src/main/scala/cognite/spark/v1/DataSetsRelation.scala

```diff
@@ -24,11 +26,22 @@ class DataSetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
 
   override def uniqueId(a: DataSet): String = a.id.toString
 
-  override def getStreams(filters: Array[Filter])(
+  override def getStreams(sparkFilters: Array[Filter])(
       client: GenericClient[IO],
       limit: Option[Int],
-      numPartitions: Int): Seq[fs2.Stream[IO, DataSet]] =
-    Seq(client.dataSets.list(limit))
+      numPartitions: Int): Seq[fs2.Stream[IO, DataSet]] = {
+    // not super necessary to do filter pushdown here, but it looks nicer when we don't fetch all datasets in Jetfire
```

Comment:
Maybe obvious what this means, but I didn't get it.. can you elaborate/rephrase?

Reply:
sorry it's probably just confusing. I was just justifying doing filter pushdown for datasets when you are quite unlikely to have many datasets in a project.